plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -93,7 +100,7 @@ static bool is_local_file_modified(obs_properties_t *props,
 	obs_property_set_visible(looping, enabled);
 	obs_property_set_visible(speed, enabled);
 	obs_property_set_visible(seekable, !enabled);
-
+	obs_property_set_visible(reconnect_delay_time, !enabled);
```

Comment:
Please re-add the blank line here between the last `obs_property_set_visible` and `return true;`.